src/definitions/z.json

```diff
@@ -0,0 +1,37 @@
+[
+  {
+    "display": "z-index",
```

Comment:
This is just a display name. The idea is to put a display name as friendly as possible, not the same as the attribute name.
My suggestion
```
name: Stak Order
description: Specifies the stack order of an element. Higher stack order gets priority over lower order in case of rendering
```